modules/jolt_physics/shapes/jolt_height_map_shape_3d.cpp

```diff
@@ -98,8 +98,8 @@ JPH::ShapeRefC JoltHeightMapShape3D::_build_height_field() const {
 			const real_t height = row[x];
 
 			// Godot has undocumented (accidental?) support for holes by passing NaN as the height value, whereas Jolt
-			// uses `FLT_MAX` instead, so we translate any NaN to `FLT_MAX` in order to be drop-in compatible.
-			row_rev[x] = Math::is_nan(height) ? FLT_MAX : (float)height;
+			// uses `std::numeric_limits<float>::max()` instead, so we translate any NaN to `std::numeric_limits<float>::max()` in order to be drop-in compatible.
```

Comment:
The first change to this comment isn't strictly true. Jolt uses `FLT_MAX`, not `std::numeric_limits<float>::max()`.

(Also, if I can nitpick, the wrapping of this comment looks a bit odd now as well.)